pandas/core/frame.py

```diff
@@ -5177,53 +5177,55 @@ def append(self, other, ignore_index=False, verify_integrity=False):
     def join(self, other, on=None, how='left', lsuffix='', rsuffix='',
              sort=False):
         """
-        Join columns with other DataFrame either on index or on a key
-        column. Efficiently Join multiple DataFrame objects by index at once by
+        Join columns with other DataFrame either on index or a key
+        column.
+
+        Efficiently Join multiple DataFrame objects by index at once by
         passing a list.
 
         Parameters
         ----------
         other : DataFrame, Series with name field set, or list of DataFrame
             Index should be similar to one of the columns in this one. If a
             Series is passed, its name attribute must be set, and that will be
-            used as the column name in the resulting joined DataFrame
+            used as the column name in the resulting joined DataFrame.
         on : name, tuple/list of names, or array-like
             Column or index level name(s) in the caller to join on the index
             in `other`, otherwise joins index-on-index. If multiple
             values given, the `other` DataFrame must have a MultiIndex. Can
             pass an array as the join key if it is not already contained in
-            the calling DataFrame. Like an Excel VLOOKUP operation
+            the calling DataFrame. Like an Excel VLOOKUP operation.
         how : {'left', 'right', 'outer', 'inner'}, default: 'left'
             How to handle the operation of the two objects.
 
-            * left: use calling frame's index (or column if on is specified)
-            * right: use other frame's index
+            * left: use calling frame's index (or column if on is specified).
```

Comment:
Understood why you did this, but generally you don't put periods at the end of bullet points